src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/OSPlatform.cs

```diff
@@ -15,7 +17,10 @@ namespace System.Runtime.InteropServices
 
         public static OSPlatform Linux { get; } = new OSPlatform("LINUX");
 
-        public static OSPlatform OSX { get; } = new OSPlatform("OSX");
+        public static OSPlatform macOS { get; } = new OSPlatform("MACOS");
+
+        [EditorBrowsable(EditorBrowsableState.Never)] // https://github.com/dotnet/runtime/issues/33331#issuecomment-650326500
+        public static OSPlatform OSX { get; } = macOS;
```

Comment:
This is a breaking change.

Reply:
how?

Reply:
For example 

```
void Foo (OSPlatform osp)
{
	if (osp.ToString() == "OSX")
		return;
}
```

Reply:
ok, then we should add the macOS/OSX magic to `ToString()` too

Reply:
> ok, then we should add the macOS/OSX magic to ToString() too

Also, Equals and number of other cases.

I do not think we should be changing the string identifier for OSX/macOS. It should stay as OSX. OSX is hardcoded in many places that are hard/impossible to fix, like NuGet RIDs. We are not gaining much by changing it ad-hoc in subset of places. If we want to change it, it should be done everywhere and it should be a feature on its own.

Reply:
We discussed this during the Design Review meeting today and got a direction on how to address this breaking change risk while still handling equivalence.  The [comments are posted on #33331](https://github.com/dotnet/runtime/issues/33331#issuecomment-656849168), but at a high-level, `OSPlatform` should not try to handle equivalence between platforms itself; instead, the `RuntimeInformation.IsOSPlatform` should handle it.